meteor/client/ui/Settings/ShowStyle/VariantSettings.tsx

```diff
@@ -79,42 +79,31 @@ export const ShowStyleVariantsSettings = withTranslation()(
 					return
 				}
 
-				_.forEach(newVariants, (entry) => {
-					MeteorCall.showstyles
-						.insertShowStyleVariantWithBlueprint(
-							entry.showStyleBaseId,
-							entry.blueprintConfig,
-							entry._id,
-							entry._rundownVersionHash,
-							entry.name
-						)
-						.catch((error) => {
-							NotificationCenter.push(
-								new Notification(
-									undefined,
-									NoticeLevel.WARNING,
-									t('Failed to import Variant ' + entry.name + '. Make sure it is not already imported.', {
-										errorMessage: error + '',
-									}),
-									'VariantSettings'
-								)
-							)
-						})
-				})
+				this.importVariantsFromArray(newVariants)
 			}
 			reader.readAsText(file)
 		}
+
+		importVariantsFromArray = (variants: ShowStyleVariant[]) => {
+			const { t } = this.props
+			_.forEach(variants, (entry: ShowStyleVariant) => {
+				MeteorCall.showstyles.insertShowStyleVariantWithBlueprint(entry, entry._id).catch(() => {
+					NotificationCenter.push(
+						new Notification(
+							undefined,
+							NoticeLevel.WARNING,
+							t('Failed to import Variant {{name}}. Make sure it is not already imported.', { name: entry.name }),
+							'VariantSettings'
+						)
+					)
+				})
+			})
+		}
+
 		copyVariant = (showStyleVariant: ShowStyleVariant) => {
-			MeteorCall.showstyles
-				.insertShowStyleVariantWithBlueprint(
-					showStyleVariant.showStyleBaseId,
-					showStyleVariant.blueprintConfig,
-					getRandomId(),
-					'',
-					'Copy of ' + showStyleVariant.name
-				)
-				.catch(console.error)
+			MeteorCall.showstyles.insertShowStyleVariantWithBlueprint(showStyleVariant).catch(console.error)
```

Comment:
Not sure if I already commented it here, but use `logger` instead of logging directly to the console

Reply:
Changed to use `logger`!